NewsBook/Application/Commands/News/NewsCommands.cs

```diff
@@ -0,0 +1,21 @@
+﻿using MediatR;
+using NewsBook.Mediator.Response;
+using NewsBook.ModelDTO.News;
+
+namespace NewsBook.Mediator.Commands.News
+{
+    public class PostNewsQuery : IRequest<NewsReadDTO>
+    {
+        public NewsWriteDTO newsDTO;
```

Comment:
Why are we using camel case here? I am only highlighting it once. Update the same scenario in all the places of the project.